<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vriddhi: Resolution Enhancer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Vridhi_icon.jpg</normaloff>Vridhi_icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(225, 246, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnEnhanceImage">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>225</y>
      <width>400</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enhance Image</string>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 193, 210);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEnhanceVideo">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>225</y>
      <width>400</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enhance Video</string>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 193, 210);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
